widgets/src/layout/stack.rs

```diff
@@ -46,16 +46,11 @@ impl Render for Stack {
     let mut size = ZERO_SIZE;
     let (ctx, children) = ctx.split_children();
     for c in children {
-      ctx.perform_child_layout(c, stack_clamp);
-      let rect = ctx.widget_box_rect(c).unwrap();
-      size = size.max(rect.max().to_vector().to_size());
+      let child_size = ctx.perform_child_layout(c, stack_clamp);
+      size = size.max(child_size);
```

Comment:
it seems the size should be expand by the position + size_box?  it seems a bit strange, if it just with red background in a top_left corner, as the example bellow:
``` 
stack! {
    background: Color::RED,
    @SizedBox {
        anchor: Anchor::left_top(100., 100.),
        size: Size::new(100., 100.),
    }
 }
 ```

Reply:
I want to keep the layout simple and consistent. If we make the size bigger towards the bottom right, why not make it bigger towards the top left if a child is in a negative position?

I set the stack's size to fit its children, ignoring their positions. This way allow their children hit and paint outside the stack.